docker/packager/packager

```diff
@@ -143,7 +143,7 @@ def parse_env_variables(build_type, compiler, sanitizer, package_type, image_typ
 
     if unbundled:
         # TODO: fix build with ENABLE_RDKAFKA
```

Comment:
FWIW unbundled build works with the unbundled kafka (1.4.0.r68.g4fea8d4c-1), worth give a try (i.e. remove `-DUSE_INTERNAL_RDKAFKA_LIBRARY=1`)?